docs/fastdds/dds_layer/core/status/status.rst

```diff
@@ -3,3 +3,630 @@
+  does not affect this count.
+
+* **alive_count_change**: The change in **alive_count** since
+  the last time |on_liveliness_changed| was called or the status was read.
+  It can have positive or negative values.
+
+* **not_alive_count_change**: The change in **not_alive_count** since
+  the last time |on_liveliness_changed| was called or the status was read.
+  It can have positive or negative values.
+
+* **last_publication_handle**: Handle to the last :ref:`dds_layer_publisher_dataWriter`
+  whose liveliness status was changed.
+  If no liveliness has ever changed, it will have value ``c_InstanceHandle_Unknown``.
+
+
+.. _dds_layer_core_status_requestedDeadlineMissedStatus:
+
+RequestedDeadlineMissedStatus
+-----------------------------
+
+This status changes every time the :ref:`dds_layer_subscriber_dataReader` did not receive
+data within the deadline period configured on its :ref:`dds_layer_subscriber_dataReaderQos`.
+
+List of status data members:
+
++--------------------------+-----------------------------------------+
+| Data Member Name         | Type                                    |
++==========================+=========================================+
+| total_count              | ``int32_t``                             |
++--------------------------+-----------------------------------------+
+| total_count_change       | ``int32_t``                             |
++--------------------------+-----------------------------------------+
+| last_instance_handle     | ``InstanceHandle_t``                    |
++--------------------------+-----------------------------------------+
+
+* **total_count**: Total cumulative count of missed deadlines for any instance read by the
+  current :ref:`dds_layer_subscriber_dataReader`.
+  As the deadline period applies to each instance of the :ref:`dds_layer_topic_topic` independently,
+  the count will will be incremented by one for each instance for which data
+  was not received in the deadline period.
+
+* **total_count_change**: The change in **total_count** since
+  the last time |on_requested_deadline_missed| was called or the status was read.
+  It can only have zero or positive values.
+
+* **last_instance_handle**: Handle to the last instance that missed the deadline.
+  If no deadline was ever missed, it will have value ``c_InstanceHandle_Unknown``.
+
+.. warning::
+
+    Currently this status is not supported and will be implemented in future releases.
+    As a result, trying to access this status will return ``NOT_SUPPORTED``
+    and the corresponding listener will never be called.
+
+
+.. _dds_layer_core_status_requestedIncompatibleQosStatus:
+
+RequestedIncompatibleQosStatus
+------------------------------
+
+This status changes every time the :ref:`dds_layer_subscriber_dataReader` finds a
+:ref:`dds_layer_publisher_dataWriter` that matches the :ref:`dds_layer_topic_topic` and has
+a common partition, but with a QoS that is incompatible with the one defined on the
```

Comment:
In general for the sub-section. Either "with a QoS configuration" or "with QoS that are"